src/CropJoin.sol

```diff
@@ -33,19 +86,16 @@ interface ERC20 {
 }
 
 // receives tokens and shares them among holders
-contract CropJoin {
-    // --- Auth ---
+contract CropJoinImp {
+    bytes32 slot0;
     mapping (address => uint256) public wards;
-    function rely(address usr) public auth { wards[usr] = 1; emit Rely(msg.sender); }
-    function deny(address usr) public auth { wards[usr] = 0; emit Deny(msg.sender); }
-    modifier auth { require(wards[msg.sender] == 1, "CropJoin/non-authed"); _; }
+    uint256 public live;
```

Comment:
These two can be private now to cut down on bytecode size since the proxy exposes both directly.

Reply:
Done.